src/tests/suite/io_conf/daos_generate_io_conf.c

```diff
@@ -326,42 +373,48 @@ struct operation operations[] = {
 	[UPDATE_ARRAY] = {
 		.op = &update_array,
 	},
+	[FETCH] = {
+		.op = &fetch_array,
+	},
 	[PUNCH_ARRAY] = {
 		.op = &punch_array,
 	},
-	[UPDATE_SINGLE] = {
-		.op = &update_single,
-	},
 	[PUNCH_AKEY] = {
 		.op = &_punch_akey,
 	},
-	[FETCH] = {
-		.op = &fetch_array,
+};
+
+struct operation single_operations[] = {
+	[UPDATE_SINGLE] = {
+	    .op = &update_single,
+	},
+	[FETCH_SINGLE] = {
+	    .op = &fetch_single,
+	},
+	[PUNCH_AKEY_SINGLE] = {
+	    .op = &_punch_akey,
 	},
 };
 
 /* Generate the ioconf
  *
- * update --epoch 1 --recx "[0, 2] [3, 8] [12, 18]"
- * update --epoch 1 --single
- * update --epoch 2 --recx "[1, 3] [5, 10] [12, 14] [100, 108]"
- * update --epoch 3 --recx "[0, 8] [13, 17] [90, 104]"
- * update --epoch 4 --recx "[1, 20] [80, 96] [110, 120]"
- * update --epoch 4 --single
- *
+ * update --tx 1 --snap --recx "[0, 2] [3, 8] [12, 18]"
+ * update --tx 2 --snap --recx "[1, 3] [5, 10] [12, 14] [100, 108]"
+ * update --tx 3 --snap --recx "[0, 8] [13, 17] [90, 104]"
+ * update --tx 4 --snap --recx "[1, 20] [80, 96] [110, 120]"
```

Comment:
Hmm, I actually thought to make snap operation independent with update. so the sequence looks like

update --tx 1  --recx ........
update --tx 2 --recx .......
snap --tx 3

more like a real use case.  But this probably will make fetch verification more complicate, we can improve it later.